layouts/404.html

```diff
@@ -27,39 +27,20 @@ <h1 class="main-heading">
 
     <div class="page-vote-404" id="main" role="main">
 
-      <a class="usa-skipnav" href="#main-content">Skip main navigation</a>
-
       <div class="bg-lightblue">
       <section class="grid-container page-inner" id="main-content">
-        <div class="grid-row">
-          <div class="tablet:grid-col-6">
-            <div lang="en">
-            <span class="usa-label">English</span>
-
-            <h2 class="usa-intro">Page Not Found</h2>
-
-            <p>We can't find the page you're looking for. It might have been removed, had its name changed, or is temporarily unavailable.</p>
-
-            <ul>
-              <li>If you typed the page URL, check the spelling.</li>
-              <li>Go to our <a href="/">main page</a> and try again.</li>
-              <li>Visit your <a href="https://www.usa.gov/election-office">state’s election office website</a> for state-wide voting guidance.</li>
-            </ul>
-          </div>
-        </div>
-
-        <div class="tablet:grid-col-6">
-          <div lang="es">
-            <span class="usa-label">Español</span>
+        <div class="grid-row grid-gap">
+          <div class="tablet:grid-col-4"></div>
+          <div class="tablet:grid-col-8">
```

Comment:
add 'right-shift' class to align the content with the page heading:
`<div class="tablet:grid-col-8 right-shift">`